### ISSUE TITLE
Testcafe page hanged while sign-in case running 

### ISSUE THREAD

**Issue by hang-ren** [DevExpress/testcafe#1580]
### Are you requesting a feature or reporting a bug?
Reporting a bug

### What is the current behavior?
Testcafe page hanged when my sign in case running.

### What is the expected behavior?
It should be successful to sign-in, or return any error, but it is hanging.

### How would you reproduce the current behavior (if this is a bug)?
Just running my case will be reproduced this bug:
testcafe chrome ./testLogin_debug.js

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:
http://ezhome-alpha-3d.shejijia.com/?brand=ezhome
Test code

```js
import { Selector } from 'testcafe';
const continueBtn = Selector('button.btn_continuebtn');
const closeWelcomeImg = Selector('img.closewelcome');
const loginLink = Selector('a.loginpp');
const loginIframe = Selector('#loginFrame');
const usrnameInput = Selector('#username');
const pwdInput = Selector('#password');
const loginButton = Selector("a.sso-submit");
const usernameText = Selector('span.nametext');


fixture `Login`
    .page `http://ezhome-alpha-3d.shejijia.com/?brand=ezhome`;

test('login_positive', async t => {
    /* Test Login */
    await t
            .maximizeWindow()
            .click(continueBtn)
            .click(closeWelcomeImg)
            .wait(2000)
            .click(loginLink)
            .switchToIframe(loginIframe)
            .wait(2000)
            .typeText(usrnameInput, '15502108088')
            .typeText(pwdInput, '123456')
            .click(loginButton)
            .switchToMainWindow()
            .wait(2000)
            .expect(usernameText.textContent).contains('爱飞SH', 'Actual username is not as expected.')
});
```
### Specify your

* operating system: Windows 7
* testcafe version:0.16.2
* node.js version:v6.11.0

**Comment by LavrovArtem** [comment]
Hi @hang-ren,
I've investigated your issue and we're already working on this. We have an issue for it in our proxy: https://github.com/DevExpress/testcafe-hammerhead/issues/1191.
We'll notify you when it's fixed.

**Comment by hang-ren** [comment]
Thanks you all.
By the way, I see the fix merged into the branch. When the next package will be released ?

**Comment by AlexanderMoskovkin** [comment]
Hi @hang-ren,

We'll provide you with a version with a fix in 1-2 days.

**Comment by hang-ren** [comment]
Thanks for the quick fix!
But also have a question that how to install the bump version or update the files which include the fix?
Is there any command need to force in my local testcafe workspace?

**Comment by miherlosev** [comment]
We've published the new `TestCafe` version - `v0.17.0-alpha1`.
To install it use the following command:
`npm install testcafe@0.17.0-alpha1`

**Comment by hang-ren** [comment]
I see, really many thanks to you guys!
